layouts/default.vue

```diff
@@ -1,4 +1,5 @@
 <script setup lang="ts">
+import ScrollTopButton from '~/components/ScrollTopButton.vue'
```

Comment:
I have no idea why it was not picking up the component like the others... I had to import it... and when I work in this component... I have to re-run nuxt to see updates. Not sure what is happening

Reply:
it worked! So, odd

Reply:
glad it works!